web/controller/components/home/ControllerHome.js

```diff
@@ -120,7 +122,11 @@ export function ControllerHome() {
         </Grid>
       </Grid>
       <div className={classes.noLicLink}>
-        <a className="fr-link" href="#">
+        <a
+          className="fr-link"
+          href="#"
+          onClick={() => setShowHoraireServiceModal(true)}
+        >
```

Comment:
:green_circle: Warning eslint ici : https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/bccf0aeab8dd337c5f134f892a6d3588fbc29bdf/docs/rules/anchor-is-valid.md
J'ai essayé en modifiant <a> par <button> mais le style est un peu changé, peut être à voir avec Audrey si on veut le fix.